Source/WebKit/WebProcess/WebPage/FindController.cpp

```diff
@@ -228,7 +228,12 @@ void FindController::updateFindUIAfterPageScroll(bool found, const String& strin
         m_findPageOverlay->setNeedsDisplay();
     }
 
-    if (found && (!options.contains(FindOptions::ShowFindIndicator) || !selectedFrame || !updateFindIndicator(*selectedFrame, shouldShowOverlay)))
+    bool wantsFindIndicator = found && options.contains(FindOptions::ShowFindIndicator);
+    bool canShowFindIndicator = selectedFrame;
```

Comment:
Is it the frame that wants the find indicator/can show it or something else?

Reply:
It's the frame containing the DOM selection (irrelevant in our case since we don't have one).